src/NHibernate/Driver/SQLite20Driver.cs

```diff
@@ -19,74 +20,85 @@ namespace NHibernate.Driver
 	/// Please check <a href="https://www.sqlite.org/">https://www.sqlite.org/</a> for more information regarding SQLite.
 	/// </para>
 	/// </remarks>
+#if DRIVER_PACKAGE
+	public class SQLiteDriver : DriverBase
+#else
+	[Obsolete("Use NHibernate.Driver.SQlLite NuGet package and SQLiteDriver."
+		+ "  There are also Loquacious configuration points: .Connection.BySQLiteDriver() and .DataBaseIntegration(x => x.SQLiteDriver()).")]
 	public class SQLite20Driver : ReflectionBasedDriver
+#endif
 	{
+#if DRIVER_PACKAGE
+		public SQLiteDriver()
```

Comment:
If we consider our naming should hint at the chosen data provider, it would then need to be renamed `SystemDataSQLiteDriver`. But personally I am more for a neutral naming like `SQLiteDriver` chosen here.